Automattic/jetpack
# Docker Image: Update Ubuntu version to 21.04

#### Changes proposed in this Pull Request:
While trying to install the Jurassic Tube scripts on an M1 Mac, I was
seeing segmentation fault errors from the `curl` commands. After some
investigation, it l[ooks to be a problem with `libcrypto`](https://github.com/docker/for-mac/issues/5846#issuecomment-881244406) and upgrading
the base image to ubuntu:21.04 fixes the issue.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Run `jetpack docker build-image`
* Check everything still functions ok
* If on an M1 Mac, run `jetpack docker exec curl https://jurassic.tube/jtuber-v0.4.sh` and check that the script is returned with this change.
